template_config.yml

```diff
@@ -0,0 +1,27 @@
+# This config represents the latest values used when running the plugin-template. Any settings that
+# were not present before running plugin-template have been added with their default values.
+
+plugin_name: pulp_maven
+plugin_app_label: maven
+plugin_snake: pulp_maven
+plugin_caps: PULP_MAVEN
+plugin_caps_short: MAVEN
+plugin_camel: PulpMaven
+plugin_camel_short: Maven
+plugin_dash: pulp-maven
+plugin_dash_short: maven
+black: True
+check_commit_message: True
+coverage: False
+deploy_client_to_pypi: True
+deploy_client_to_rubygems: True
+deploy_daily_client_to_pypi: True
+deploy_daily_client_to_rubygems: True
+deploy_to_pypi: True
+docs_test: True
+pydocstyle: True
+test_bindings: False
+pypi_username: None
```

Comment:
Is it correct that the username is None?

Reply:
No, I need to correct this.